build-parent/pom.xml

```diff
@@ -53,6 +53,11 @@
         <!-- Enable APT by default for Eclipse -->
         <m2e.apt.activation>jdt_apt</m2e.apt.activation>
 
+        <!-- These 2 properties are used in catalog-overrides.json -->
+        <!-- Supported Java versions, interpreted as a version range -->
+        <supported-java-versions>[11,)</supported-java-versions>
```

Comment:
Maybe reuse `maven.compiler.release` from the root pom.xml?

Reply:
Good idea, done!